package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "webpack",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Adanzeakonobi/webpack.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Adanzeakonobi/webpack/issues"
+  },
+  "homepage": "https://github.com/Adanzeakonobi/webpack#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^6.1.12",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.72.0",
+    "webpack-cli": "^4.9.2"
```

Comment:
- [ ] As it is a requirement for this project, please make sure you have `Webpack dev server`  set up and running with no errors as it will enable to run (`serve`) your app. You can add the `devDependency` here like so: `"webpack-dev-server": "^4.7.4"`, and then you update the `scripts` section like so: ` "start": "webpack serve --open"`, now you will be able to run your app with the `npm start` command.